plugins/catalog-backend/src/database/Database.test.ts

```diff
@@ -74,7 +76,10 @@ describe('Database', () => {
           name: 'c',
           namespace: 'd',
           labels: { e: 'f' },
-          annotations: { g: 'h' },
+          annotations: {
+            g: 'h',
+            'backstage.io/managed-by-location': undefined,
```

Comment:
unfortunately you won't be able to do this, these are strictly strings

Reply:
- [x] remove key instead of having it with value `undefined`